UserDetailsClient/UserDetailsClient.iOS/UserDetailsClient.iOS.csproj

```diff
@@ -108,8 +109,20 @@
     <Reference Include="Microsoft.Identity.Client, Version=3.0.8.0, Culture=neutral, PublicKeyToken=0a613f4dd989e8ae, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Identity.Client.3.0.8\lib\xamarinios10\Microsoft.Identity.Client.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.IdentityModel.JsonWebTokens, Version=5.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Why do you need this? We are trying to discourage people from parsing tokens as: 

- MSAL takes care of all the details around expiration and UPN
- Parsing the token should also involve some cryptographic validation. This is not easy to get right.